api-server/Dockerfile

```diff
@@ -1,15 +1,16 @@
-FROM node:gallium
+FROM node:19-alpine
 
-ADD ./ /api-server
 WORKDIR /api-server
 
-RUN chmod +x ./docker-scripts/start.sh
-RUN chmod +x ./docker-scripts/wait-for-it.sh
-RUN npm install -g npm@8.7.0
+RUN apk -U --no-cache add \
+  bash just
 
-RUN npm ci
-RUN npm run build
+COPY package.json package-lock.json ./
```

Comment:
won't this be copied with COPY . . ?

Reply:
no - because there is a .dockerignore directive. the reason to separate these is to make use of the docker build cache more effectively - not refetch dependencies every time a minor edit is made on source code.